processor/src/main/java/com/linecorp/decaton/processor/runtime/internal/CompletionImpl.java

```diff
@@ -41,6 +41,12 @@ public static CompletionImpl completedCompletion() {
         return comp;
     }
 
+    static CompletionImpl failedCompletion(Throwable ex) {
+        CompletionImpl comp = new CompletionImpl();
+        comp.completeExceptionally(ex);
```

Comment:
Hm, since `completeExceptionally` is not expected to be used from outside of the class, let's just completing the underlying CompletableFuture directly?

i.e. `comp.future.completeExceptionally(ex)`